compiler_expr.go

```diff
@@ -1266,6 +1268,9 @@ func (e *compiledLiteral) emitGetter(putOnStack bool) {
 }
 
 func (e *compiledLiteral) constant() bool {
+	if _, ok := e.val.(valueBigInt); ok {
+		return false
```

Comment:
Why is it not constant?

Reply:
Since the exception TypeError Cannot mix BigInt and other types is thrown during compilation when mixed types are used in operations.

Reply:
There is [support](https://github.com/dop251/goja/blob/8130cadc5774d6c30ecbef6e9cb07c406afcdcc5/compiler_expr.go#L2468) for it, the only thing that needs to be added is `RangeError` [here](https://github.com/dop251/goja/blob/8130cadc5774d6c30ecbef6e9cb07c406afcdcc5/compiler_expr.go#L2449).